Add tips for ongoing learning to launching & landings episode (#1342)
carpentries/instructor-training: _episodes/23-introductions.md

> > - Plan next steps. Does the local community have resources to support continued learning? 
> > Do you have advice for how learners might continue on their own? Even
> > if you have no advice, asking learners to take a moment to discuss their own plans can support them in taking a next step sooner rather than later.
> > - Reiterate where the lesson materials can be found, and encourage them to apply at least one of the skills to their own work within the next few days.  Some examples could be:

> > >    - Recreate an spreadsheet from a past presentation in R

Reviewer: What do we mean with "presentation"? maybe "analysis" fits better?
```suggestion
> >    - Recreate an spreadsheet from a past analysis in R
```

> >    - Write a lab notebook entry in R-markdown

> > >    - Backup a thesis or manuscript by storing it on a version control server

Reviewer: Some version control systems need of a server - e.g., SVN, CVS - but with git and mercurial that's not needed. So I imagine it would be better if we say "system" rather than "server".
```suggestion
> >    - Backup a thesis or manuscript by storing it on a version control system
```
However, if we change that for system, then it's [not a backup any more, as it would be in our computer](https://swcarpentry.github.io/git-novice/08-collab/index.html#version-history-backup-and-version-control).

Maybe a rewrite of the sentence as:
```suggestion
> >    - Backup a thesis or manuscript by storing it on a version control system in a remote server;
```
could be better? though may make it look way more difficult than what it is.

> > >    - Backup a thesis or manuscript by storing it on a version control server

Author: close to what you sugggested, but a bit further rewrite

> > > - Reiterate where the lesson materials can be found, and encourage them to apply at least one of the skills to their own work within the next few days.  Some examples could be:
> > >    - Recreate an spreadsheet from a past analysis in R
> > >    - Write a lab notebook entry in R-markdown
> > >    - Backup a thesis or manuscript by storing it on a remote server with a version control system
> > >    - Log in to a remote machine and run an analysis there

Author: ```suggestion
> > - Reiterate where the lesson materials can be found, and encourage them to apply at least one of the skills to their own work within the next few days.  Potential examples include:
> >    - Recreate an spreadsheet from a past analysis in R;
> >    - Write a lab notebook entry in R-markdown;
> >    - Backup a thesis or manuscript by storing it on a remote server with a version control system; and
> >    - Log in to a remote machine and run an analysis there.
```
